src/main/resources/default/templates/biz/jobs/interactive-job.html.pasta

```diff
@@ -13,84 +17,73 @@
         </li>
     </i:block>
 
-    <w:pageHeader>
-        <div class="row">
-            <div class="col-md-8">
-                @job.getCurrentLabel(context)
+    <t:pageHeader>
+        <i:block name="title">@job.getCurrentLabel(context)</i:block>
+        <t:iconInlineInfo icon="fa fa-info-circle">@job.getCurrentDescription(context)</t:iconInlineInfo>
+        <i:block name="actions">
+            <i:if test="!job.getJobInfos().isEmpty()">
+                <a class="btn btn-outline-link" href="/jobs/infos/@job.getName()">
+                    <span><i class="fa fa-book"></i></span> @i18n("JobsController.showInfos")
+                </a>
+            </i:if>
+        </i:block>
+    </t:pageHeader>
+
+    <div class="row">
+        <i:if test="hasVisibleParameters">
+            <div class="col-md-3">
+                <t:editForm url="@apply('/job/%s', job.getName())">
+                    <div class="well">
```

Comment:
I don't think "well" is doing anything in tycho anymore

Reply:
-> migrate to `card`

Reply:
I now removed the div instead of migrating it in order to mimic the look of standard jobs pages.